### ISSUE TITLE
vpk_linux32 segfault

### ISSUE THREAD

**Issue by vmarkovtsev** [Jorl17/half-life-source-hd-sound-fix#2]
```
extracting materials/models/agibs/a_gib.vmt

Thread 1 "vpk_linux32" received signal SIGSEGV, Segmentation fault.
#0  0x08077540 in CPackedStoreReadCache::ReadCacheLine(FileHandleTracker_t&, CachedVPKRead_t&) ()
#1  0x0807bf42 in CPackedStoreReadCache::BCanSatisfyFromReadCacheInternal(unsigned char*, CPackedStoreFileHandle&, FileHandleTracker_t&, int, int, int&) ()
#2  0x0807c116 in CPackedStoreReadCache::BCanSatisfyFromReadCache(unsigned char*, CPackedStoreFileHandle&, FileHandleTracker_t&, int, int, int&) ()
#3  0x0807c24b in CPackedStore::ReadData(CPackedStoreFileHandle&, void*, int) ()
#4  0x0804dfd2 in main ()
```

Ubuntu 16.04 x64

**Comment by vmarkovtsev** [comment]
Should use https://bitbucket.org/panzi/unvpk

**Comment by Jorl17** [comment]
I currently do to have much time to work on this. Perhaps if you could try to submit a pull request?

I will try to find some time in the weekend but I can't really promise anything.

**Comment by vmarkovtsev** [comment]
Sure! Just need to take time as I cannot launch HL at work to check everything :)

**Comment by Jorl17** [comment]
Fixed by fc74e8342e49481a42b244ebe171831df7ddf89d.
